design/cdn-edge.md

```diff
@@ -6,62 +6,62 @@ On this diagram the arrow points in the direction of who instantiated the reques
 
 ## Internet
 
-The internet is where all incoming requests will come from. When a viewer requests a video segment from us this is where the request lifecycle starts.
+The internet is where all incoming requests will come from. When a viewer requests a video segment from us, this is where the request lifecycle starts.
 
 ## Layer 1
 
-Layer 1 is the first port of entry, all users will talk to a layer 1 cache server. 
+Layer 1 is the first port of entry; all users will talk to a Layer 1 cache server.
 
-Layer 1 cache servers will have very high outgoing bandwidth. This is because most if not all of the files we serve will be cache hits, since everyone gets the same video files. However we will have to serve a large amount of people therefore we will need a lot of bandwidth at the edge of our network.
+Layer 1 cache servers will have very high outgoing bandwidth. This is because most, if not all, of the files we serve will be cache hits since everyone gets identical video files. However, we will have to do a large number of people; therefore, we will need a lot of bandwidth at the edge of our network.
```

Comment:
`However, we will have to do a large number of people; therefore, we will need a lot of bandwidth at the edge of our network.`

Doesnt read correctly.

`However, we will have to serve a large number of people; therefore, we will need a lot of bandwidth at the edge of our network.`